core/src/test/java/org/elasticsearch/search/simple/SimpleSearchIT.java

```diff
@@ -109,6 +108,11 @@ public void testSimpleIp() throws Exception {
         assertHitCount(search, 1L);
     }
 
+    @Override
+    protected int numberOfShards() {
```

Comment:
Because I forgot to remove that overridden method after debugging a failing test ;)